radicle-repl/src/Radicle/Repl/Capabilities.hs

```diff
@@ -2,14 +2,14 @@
 --
 -- The intent is that any set of primops may wear on their sleaves (i.e.
 -- constraints) what effects they do.
-module Radicle.Internal.Effects.Capabilities where
+module Radicle.Repl.Capabilities where
 
 import           Protolude
 
 import qualified Data.ByteString as BS
 import           Data.Text.Prettyprint.Doc (PageWidth)
 import           Data.Time
-import           Paths_radicle
+import           Paths_radicle_repl
```

Comment:
Not so sure about this change. I think this was for a call to `getDataDir` which I believe is actually the directory above. Need to look into that

Reply:
True, `findModule` will try to resolve to the top-level `rad` dir. Perhaps this could be moved to the `Radicle.Internal.CLI` module, which isn't so much about CLI but about paths (so could be renamed to ... `Paths`)?

Reply:
Oh wait, does that create circular dependencies?